### ISSUE TITLE
Unable to see app full name

### ISSUE THREAD

**Issue by ppvnf** [martinet101/WingetUI#193]
I am unable to see the app full name in the list and in info, despite it being available in Winget-cli:

- **Image 1 (list):**

![Captura de tela 2022-10-03 203134](https://user-images.githubusercontent.com/73800734/193703969-c1120f17-142a-476c-a797-9be39a06b252.png)

- **Image 2 (info):**

![Captura de tela 2022-10-03 203305](https://user-images.githubusercontent.com/73800734/193704037-1ec63c6e-fd09-41a0-b095-ffe1f0f32af1.png)

- **Image 3 (Powershell):**

![Captura de tela 2022-10-03 203401](https://user-images.githubusercontent.com/73800734/193704103-02ff4874-f09c-4425-8881-4f7925c45798.png)

**Comment by ppvnf** [comment]
About image 1, what do you think about making the tooltip show the full name? 
About image 2, I see 2 ways to solve that: reduce font size or make it possible to select text that is off screen

**Comment by martinet101** [comment]
Well, this text cut out must be fixed

**Comment by martinet101** [comment]
> despite it being available in Winget-cli

Well, running `winget search` does not always show the full app names

**Comment by ppvnf** [comment]
> > despite it being available in Winget-cli
> 
> Well, running `winget search` does not always show the full app names

It puzzles me why Winget-cli sometimes doesn't show the full app name. I would open an issue in Winget-cli, but I don't know how to explain it very well. I am aware there's not much you can do to mitigate this issue in WingetUI, but thanks for still trying!

**Comment by martinet101** [comment]
> It puzzles me why Winget-cli sometimes doesn't show the full app name. I would open an issue in Winget-cli, but I don't know how to explain it very well. I am aware there's not much you can do to mitigate this issue in WingetUI, but thanks for still trying!

yeah, winget does behave weirdly sometimes (i gues we can blame microsoft for it hehe)
